packages/examples/cvat/recording-oracle/tests/unit/modules/webhook/test_process_intermediate_results.py

```diff
@@ -0,0 +1,7 @@
+from src.modules.webhook.process_intermediate_results import (
+    process_image_label_binary_intermediate_results,
+)
+
+
+def test_process_image_label_binary_intermediate_results(intermediate_results):
+    results = process_image_label_binary_intermediate_results(intermediate_results)
```

Comment:
What does this test do?

Reply:
It just checks if the function runs without errors.

If you want, I can extend it to test if it is also consistent with the desired output format (structure and types).

Checking the returned values (agreement etc.) did not seem to make much sense, as the example data is not designed to produce specific agreement values and the functions calculating the agreement values are tested elsewhere.

Reply:
Yes, I think it would be good to check the structure and types.

Reply:
Agree with @flopez7, the rest looks fine.